README.md

```diff
@@ -10,24 +10,35 @@ The repository is organized by products:
 
  - [AI Platform Training](training)
  - [AI Platform Prediction](prediction)
+ 
+<!--
+ Commenting these out until we have at least a sample for them in the repo:
  - [AI Platform Notebooks](notebooks)
  - [Deep Learning VM Images](dlvm)
  - [Data Label Services](https://cloud.google.com/data-labeling/docs/)
  - [Built-in Algorithms](built_in_algorithms)
- 
+ -->
 
 ## Getting Started
 
-If this is your first time using [AI Platform](https://cloud.google.com/ml-engine/docs/), we suggest you take a look at the [Introduction to AI Platform](https://cloud.google.com/ml-engine/docs/technical-overview) docs to get started.
+We highly recommend that you start with our [Quick Start Sample](./quickstart).
 
 ## Navigating this Repository
 
-In order to run any of the samples in this repository, you'll need to setup your environment first.
```

Comment:
Does it makes sense to mention other frameworks here?

Reply:
Done.